src/Core/PluginCollection.php

```diff
@@ -229,12 +229,20 @@ public function get(string $name): PluginInterface
      * @param array<string, mixed> $config Configuration options for the plugin.
      * @return \Cake\Core\PluginInterface
      * @throws \Cake\Core\Exception\MissingPluginException When plugin instance could not be created.
+     * @throws \InvalidArgumentException When class name cannot be found.
+     * @phpstan-param string|class-string<\Cake\Core\PluginInterface> $name
```

Comment:
Please use `@psalm-*` prefixed annotations for consistency. Phpstan reads and supports those too.